packages/ruleset/src/rules/property-attributes.js

```diff
@@ -0,0 +1,15 @@
+const { oas3 } = require('@stoplight/spectral-formats');
+const { propertyAttributes } = require('../functions');
+const { schemas } = require('../collections');
+
+module.exports = {
+  description: 'Schema must use valid combination of type and format',
```

Comment:
Copy and paste error for the description?

Reply:
Nah, this was just a test... you passed!   LOL

Reply:
done